src/payment/strategies/braintree/braintree-hosted-form.ts

```diff
@@ -26,15 +26,21 @@ export default class BraintreeHostedForm {
         private _braintreeSDKCreator: BraintreeSDKCreator
     ) {}
 
-    async initialize(options: BraintreeFormOptions): Promise<void> {
+    async initialize(options: BraintreeFormOptions): Promise<boolean> {
```

Comment:
I think that the change of the interface is not beneficial here, as we are able to define whether we are able to initialize the form or not.